cmd/zarb/start.go

```diff
@@ -38,6 +40,10 @@ func Start() func(c *cli.Cmd) {
 			Name: "a auth",
 			Desc: "Passphrase of the key file",
 		})
+		pprofOpt := c.String(cli.StringOpt{
```

Comment:
Why not boolean. Default set to false

Reply:
it's not recommended to Expose this server Online so its better to let user expose it on any interface he/she likes by providing that interface specific address

by default if this address not supplied server will not expose any data to outside